MVCapplication/src/main/webapp/WEB-INF/resources/js/shopping_list.js

```diff
@@ -21,9 +21,8 @@ function edit(prodId, val, index) {
             var amount = $("#am" + index);
             amount.text(data);
         },
-        error: function (jqXHR, exception) {
-            console.log(exception);
-            console.log(jqXHR);
+        error: function () {
+            alert("Something went wrong. Please, try again.")
```

Comment:
Actually it's redundant, because this scenario never happens. If there's an error, server redirects to the error page.